### PR TITLE
Expose metering configuration to api

### PR DESCRIPTION
**Pull request by mate4st**
**What this PR does / why we need it**:
Exposes metering seed configuration to the api

```release-note
NONE
```

### CODE REVIEW

File: pkg/api/v2/types.go [hunk 1]
```diff
@@ -156,13 +156,25 @@ type AlertmanagerSpec struct {
...
+	// the Seed level metering settings
+	Metering Metering `json:"metering"`
 }
 
 type MLA struct {
 	// whether the user cluster MLA (Monitoring, Logging & Alerting) stack is enabled in the seed
 	UserClusterMLAEnabled bool `json:"user_cluster_mla_enabled"`
 }
 
+type Metering struct {
```
Comment (moadqassem) [inline pkg/api/v2/types.go]: Why creating the new config, can't you reuse the same one from the metering package?

Comment (mate4st) [reply pkg/api/v2/types.go]: Yes i agree

**Review by moadqassem — COMMENTED** [review]
/approve
/lgtm

**Comment by xrstf** [comment]
/hold

temporarily

**Comment by xrstf** [comment]
/hold

temporarily

**Pull request merged**